containers/layouts/LayoutsSection.js

```diff
@@ -55,6 +55,9 @@ const LayoutsSection = () => {
     };
 
     const handleChangeViewMode = async (mode) => {
+        if (mode !== VIEW_MODE.GROUP) {
```

Comment:
Why not using a truly condition?

Reply:
This is my reasoning that I thought about for a second before writing the code:
The logic as I understand is, sticky labels can only be enabled when view mode is set to group, so if we add new view mode that also does not support sticky labels, instead of adding OR clause we wouldn't need to change anything. I can change it to truthy condition if that's not a case.